c++/src/kj/async-io-win32.c++

```diff
@@ -1114,6 +1114,16 @@ private:
 Promise<void> AsyncInputStream::read(void* buffer, size_t bytes) {
   return read(buffer, bytes, bytes).then([](size_t) {});
 }
+Promise<size_t> AsyncInputStream::read(void* buffer, size_t minBytes, size_t maxBytes) {
```

Comment:
This spooked me, too, because I was also able to get your original async-win32 branch compiling by itself in MSVC, but when I based the work on master, suddenly this cropped up! My best guess is that 559044bf tickled the dead code path elimination routines in the compiler to suddenly reveal the lack of `read`. That's just a guess, though.